doc/classes/ProjectSettings.xml

```diff
@@ -231,9 +231,6 @@
 		<member name="audio/mix_rate" type="int" setter="" getter="">
 			Mixing rate used for audio. In general, it's better to not touch this and leave it to the host operating system.
 		</member>
-		<member name="audio/output_latency" type="int" setter="" getter="">
```

Comment:
That one is also a platform-specific bug it seems, normally this setting should be properly registered on all platforms where `doctool` runs, so it should be kept.